tools/automation/cache/waitForCache.rc

```diff
@@ -0,0 +1,13 @@
+<ruby>
+require "msf/core/modules/metadata/store"
+
+UserMetaDataFile = Msf::Modules::Metadata::Store::UserMetaDataFile
+store_dir = File.join(Msf::Config.config_directory, "store")
+cache_file = File.join(store_dir, UserMetaDataFile)
+framework.modules.refresh_cache_from_module_files
+while true do
```

Comment:
I assume it's up to the caller (jenkins/github actions) to have their own timeout mechanism in place? 👀

Reply:
This is actually just a copy of what we already do in automation, the break condition in this loop is consistent.